guide/getting-started/why-design-for-bitcoin.md

```diff
@@ -37,35 +37,37 @@ Illustration sources
+Few open designers currently work full-time on Bitcoin. Most have 9–5 jobs, and like bitcoin’s most open-source developers, contribute when they can. As the ecosystem grows and more investors enter the space, more opportunities to be one of the first full-time bitcoin designers are bound to open up.
+
+So if you’re looking for the next big thing in design, you can stop.
 
 ## Design for your needs
 
-Many bitcoin projects start by identifying an unmet need. Unlike traditional finance, the barriers to entry are shallow. So if you were ever unhappy with the service of your bank or financial provider, here's your chance to shape something better.
+Many design projects start by identifying an unmet need then designing something that meets it. Unlike traditional finance, the barriers to entry into bitcoin are shallow. So if you feel like your needs aren’t being met by your bank or service provider, here’s your chance to shape something better.
 
 ## Work in public
 
-Bitcoin is open-source, and designing for Bitcoin allows you also work on the same ideas of publicly sharing and collaborating on design information. Free from agencies and corporations' structures, open collaboration allows every participant to have a voice and shape the way they work based on their interests and preferences. Find out more about [open design]({{ '/guide/getting-started/open-design/' | relative_url }}).
+Bitcoin development is open-source. Designing for bitcoin is no different. It lets you learn, grow, and exchange ideas in the same open and collaborative way. Learn out more about bitcoin’s [open design]({{ '/guide/getting-started/open-design/' | relative_url }}) philosophy.
 
 ## Design money
 
-Money is a powerful force and therefore heavily regulated and controlled. The composition of bitcoin allows us to trust in its value and availability while also providing vast freedom to experiment and build on top of it. The emergence of thousands of so-called alternative coins, hundreds of wallets, and the many bitcoin businesses are the most apparent signifier of just how open this space is.
+Central authorities tightly control Government-issued currencies. This means that they are anything but open and experimental. At every level, bitcoin places a great deal of value on openness and experimentation, letting designers build the future of money in their own way. The emergence of thousands of “alternative” coins, hundreds of wallets, and countless Bitcoin businesses signify just how open this space is.
 
 ## Join a global community
 
-The doors are wide open, and Bitcoin design is discussed publicly online and any social media service you can think of. Design and build with friends and strangers, participate in the various communities and help shape an ecosystem for design and creativity.
+Design and build experiences with friends and strangers. Participate in or even help lead one of the fastest growing design ecosystems in the world.
```

Comment:
`one of the fastest growing design ecosystems in the world` I know this is just meant to sound impressive, but I find it a but fluffy and would prefer something that is more welcoming and about community.

Reply:
What's more welcoming than participating and lead in one of the fastest-growing design ecosystems?

Reply:
Is it one of the fastest-growing design ecosystems? I see way more design stuff happening in other ecosystems.